addons/l10n_sa/data/res_country_group_data.xml

```diff
@@ -0,0 +1,8 @@
+<odoo>
+    <data noupdate="1">
+        <record id="l10n_sa_country_group_gcc" model="res.country.group">
+            <field name="name">GCC</field>
```

Comment:
"GNU Compiler Selection" ? :o 

Why isn't this group declared in base ? What does it represent ? Is it something that only makes sense in Saudi Arabia ?

Reply:
Gulf Countries.  Makes sense to me, but maybe we could put it in base.

Reply:
I moved this record into the base module